[quant][pt2e] Add support for view_copy.default in linear pattern

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #98802

Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
[quant][pt2e] Add support for view_copy.default in linear pattern

Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][pt2e] Add support for view_copy.default in linear pattern"


Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo reviewed (approved):
a test would be good

jerryzh168 committed:
Update on "[quant][pt2e] Add support for view_copy.default in linear pattern"


Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][pt2e] Add support for view_copy.default in linear pattern"


Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
> a test would be good

yeah, I think we'll have more exhaustive tests after we implement the pattern matching related things properly, there are a few patterns linear can have that Kimish has listed in his notebooks

jerryzh168 committed:
Update on "[quant][pt2e] Add support for view_copy.default in linear pattern"


Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][pt2e] Add support for view_copy.default in linear pattern"


Summary:
att

Test Plan:
tested in internal models

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
sorry I just found that this op is not part of the linear pattern for the model I'm looking at, this is a op that comes from torch.flatten actually, will close this PR and submit another one
